fmriprep/workflows/bold/base.py

```diff
@@ -923,10 +924,12 @@ def init_func_preproc_wf(
     # SURFACES ##################################################################################
     surface_spaces = [space for space in output_spaces.keys() if space.startswith('fs')]
     if freesurfer and surface_spaces:
+        fslr_density = output_spaces.get('fsLR', {}).get('den', '32k')
```

Comment:
We'll need to address #1895 if we want this to work.

Reply:
i've opened poldracklab/smriprep#145 - since that isn't exclusive to this I'll submit a fix separate from this. A quick workaround is to specify at least one parameter for `fsLR`/`fsaverage`